client/src/components/interactive/stickerRenderHelpers.tsx

```diff
@@ -110,7 +116,9 @@ const MoveableSticker: React.FC<StickerRenderProps> = ({
       data.y / containerBounds.height,
     ];
     roomID && updateStickerInstancePos(roomID, id, newPos);
+    setCurrPos(newPos);
```

Comment:
I made this change after noticing the following behavior when THING was open for a while and becoming slower:

- Dragged a sticker to a new location
- When releasing mouse click, the sticker would immediately start to animate back to the original position
- Once the Firebase update was applied, the sticker would re-render back into the new location